MantidQt/CustomInterfaces/test/ALCBaselineModellingModelTest.h

```diff
@@ -144,4 +145,65 @@ class ALCBaselineModellingModelTest : public CxxTest::TestSuite {
   }
 };
 
+class ALCBaselineModellingModelTestPerformance : public CxxTest::TestSuite {
+  // ALCBaselineModellingModelPerformance *m_model;
+
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static ALCBaselineModellingModelTestPerformance *createSuite() {
+    return new ALCBaselineModellingModelTestPerformance();
+  }
+  static void destroySuite(ALCBaselineModellingModelTestPerformance *suite) {
+    delete suite;
+  }
+
+  ALCBaselineModellingModelTestPerformance() {
+    FrameworkManager::Instance(); // To make sure everything is initialized
+  }
+  void setUp() override {
+    m_model = new ALCBaselineModellingModel();
+    xTestData = {10.0, 1.0, 1.41, 10.0, 10.0, 1.73, 2.0, 2.5, 10.0};
+    yTestData = {100, 1, 2, 100, 100, 3, 4, 5, 100};
+    eTestData = {1, 2, 3, 4, 5, 6, 7, 8, 9};
+  }
+
+  void tearDown() override { delete m_model; }
+
+  void test_setData() {
```

Comment:
What are the runtimes of `test_setData` and `test_fit`? The test data is so tiny that I suspect it is not meaningful?